src/server/wsServer.ts

```diff
@@ -178,15 +182,21 @@ export class WSServer extends Disposable {
 				type: 'ws',
 				err: err,
 			});
-			console.log(`Unknown error: ${err}`);
+			console.log(localize('unknownError', 'Unknown error: {0}', err));

```

Comment:
if this is something that only you, as the maintainer, needs to see, you can keep it in English. However, if you want your users to see this, then leave it localized.